CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## Current Develop Branch
 
 ## 8.0.7 Fri Aug 07 2020
+- [#9065](https://github.com/MetaMask/metamask-extension/pull/9065): Change title of "Reveal Seed Words" page to "Reveal Seed Phrase"
+- [#8974](https://github.com/MetaMask/metamask-extension/pull/8974): Add tooltip to copy button for contacts and seed phrase
+- [#9063](https://github.com/MetaMask/metamask-extension/pull/9063): Fix broken UI upon failed password validation
+- [#9075](https://github.com/MetaMask/metamask-extension/pull/9075): Fix shifted popup notification when browser is in fullscreen on macOS
+- [#9085](https://github.com/MetaMask/metamask-extension/pull/9085): Support longer text in network dropdown
+- [#8873](https://github.com/MetaMask/metamask-extension/pull/8873): Fix onboarding bug where user can be asked to verify seed phrase twice
+- [#9104](https://github.com/MetaMask/metamask-extension/pull/9104): Replace "Email us" button with "Contact us" button
+- [#9137](https://github.com/MetaMask/metamask-extension/pull/9137): Fix bug where `accountsChanged` events stop after a dapp connection is closed.
```

Comment:
```suggestion
- [#9137](https://github.com/MetaMask/metamask-extension/pull/9137): Fix bug where `accountsChanged` events stopped for all open instances of a dapp if one was closed
```

Reply:
🤔 
This wording doesn't seem applicable to iframes though - the graph iframe we saw isn't exactly an "instance of a dapp"

Reply:
I guess I'll leave it as-is for now